src/CFNetwork/CFHTTPMessage.cs

```diff
@@ -333,7 +333,7 @@ internal static IntPtr GetAuthScheme (AuthenticationScheme scheme)
 				return _AuthenticationSchemeDigest.Handle;
 			case AuthenticationScheme.OAuth1:
 				if (_AuthenticationSchemeOAuth1 == null)
-					throw new NotSupportedException ("requires iOS 7.0 or OSX 10.9");
+					throw new NotSupportedException ("requires iOS 7.0 or OSX 10.9 and lower than iOS 12 or OSX 10.4");
```

Comment:
I assume you mean 10.14? Also if it is indeed 10.14 then it's macOS not OS X.

Note: while you're changing this please start the sentence with a capital letter and end with a dot. Also let's make it "OS X", not attached.

Reply:
"requires iOS 7.0 or macOS 10.9 and lower than iOS 12 or macOS 10.14" ?